django/db/models/sql/compiler.py

```diff
@@ -1503,7 +1503,8 @@ def as_sql(self):
         """
         sql, params = [], []
         for annotation in self.query.annotation_select.values():
-            ann_sql, ann_params = self.compile(annotation, select_format=FORCE)
+            ann_sql, ann_params = self.compile(annotation)
+            ann_sql, ann_params = annotation.select_format(self, ann_sql, ann_params, False)
```

Comment:
I suspect this line is not required or at least is untested because all passing `subquery=False` disables all the `select_format` except for the `Exists` one.

Reply:
I confirm this is required, without it `gis_tests.geoapp.tests.GeoQuerySetTest` fails on most GIS backends.